includes/widgets/video.php

```diff
@@ -314,21 +314,14 @@ protected function register_controls() {
 			'autoplay',
 			[
 				'label' => esc_html__( 'Autoplay', 'elementor' ),
+				'description' => sprintf(
+					esc_html__( 'Note: Autoplay is affected by %s', 'elementor' ),
+					'<a href="https://developers.google.com/web/updates/2017/09/autoplay-policy-changes" target="_blank">' .
+					esc_html__( 'Google’s Autoplay policy', 'elementor' ) . '</a>' .
+					esc_html__( ' on Chrome browsers.', 'elementor' )
+				),
```

Comment:
```suggestion
				'description' => sprintf(
					/* translators: 1: `<a>` opening tag, 2: `</a>` closing tag. */
					esc_html__( 'Note: Autoplay is affected by %1$s Google’s Autoplay policy %2$s on Chrome browsers.', 'elementor' ),
					'<a href="https://developers.google.com/web/updates/2017/09/autoplay-policy-changes" target="_blank">',
					'</a>'
				),
```

A single string is preferred.

Also, we should provide translators comments.